lib/cadet_web/controllers/assessments_controller.ex

```diff
@@ -17,6 +17,18 @@ defmodule CadetWeb.AssessmentsController do
     end
   end
 
+  def unsubmit(conn, %{"submissionid" => submission_id}) when is_ecto_id(submission_id) do
```

Comment:
Also, please add another `unsubmit/2` for when the user does not provide `submissionid`